### DISCUSSION TITLE
Welcome to gitcord-bot Discussions!

### DISCUSSION THREAD

**Discussion by dtemir** [MLH-Fellowship/gitcord-bot#89] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by srajan-kiyotaka** [comment]
**So I guess I will start,**
 **Hi,**
Myself **Srajan Chourasia**, 
I'm a third-year BTech Student at **IIT Goa**.
I am the current **General Secretary of Technical Affairs of IIT Goa**,
I am interested in the field of **ML, especially in Deep Learning, GAN and IoT**.
**_I was hoping to enhance my knowledge, network, contribute to others, help others, and get collaboration for my college and club,
with the help of this community_**.
_you can know more about me through my GitHub account!_
